Etch-a-Sketch: Be more explicit with instruction to use flexbox (#26766)
TheOdinProject/curriculum: foundations/javascript_basics/project_etch_a_sketch.md

2.  Create a webpage with a 16x16 grid of square divs.
    *   Create the divs using JavaScript. Don't try making them by hand with copy and pasting in your HTML file!
    *   It's best to put your grid squares inside another "container" div \(which can go directly in your HTML\).

>     *   Use flexbox to make the divs appear as a grid \(versus just one on each line\). Despite the name, do not be tempted to research/use CSS Grid for this as Grid will be taught later after foundations. This is an opportunity specifically to get more practice in for flexbox! You will get opportunities to practice CSS Grid when the time is right.

Reviewer: Suggested wording changes: 
```suggestion
    *   Use flexbox to make the divs appear as a grid \(versus just one on each line\). Despite the name, do not be tempted to research/use CSS Grid for this as Grid will be taught later after the foundations course. This is an opportunity specifically to get more practice in for flexbox!
```
